src/mixed-reality/README.rst

```diff
@@ -0,0 +1,9 @@
+# Azure CLI Mixed Reality Extension #
```

Comment:
@williexu, how does this look?

Reply:
Sure. The file name comes from https://github.com/Azure/azure-cli/blob/master/doc/extensions/authoring.md.